webcompat/webhooks/helpers.py

```diff
@@ -230,3 +234,35 @@ def prepare_rejected_issue():
     payload_request['state'] = 'closed'
     payload_request['milestone'] = invalid_id
     return payload_request
+
+
+def make_classification_request(issue_number):
```

Comment:
I wonder if we should put the classification functions in a new file: `webcompat/webhooks/ml.py`?

Just in case in the future, it grows with more functions so we do not end up with a helpers being huge. Maybe it doesn't matter. Opinions?

Reply:
Yeah, probably a good idea, I'll move it